napari/viewer.py

```diff
@@ -88,6 +88,39 @@ def update_console(self, variables):
             return
         self.window._qt_viewer.console.push(variables)
 
+    def export_figure(
```

Comment:
Same as other comment should change here.

Reply:
Also type annotations should be added